hrpc/mutate.go

```diff
@@ -494,7 +497,8 @@ var durabilities = []*pb.MutationProto_Durability{
 
 func (m *Mutate) toProto(isCellblocks bool, cbs [][]byte) (*pb.MutateRequest, [][]byte, uint32) {
 	var ts *uint64
-	if m.timestamp != MaxTimestamp {
```

Comment:
I think this was intentional behavior, though it looks odd. `m.timestamp` is set to MaxTimestamp (MaxUint64) by default during instantiation, which is meant to be interpreted as unset. So, that's why this code is not setting the pb value.
I think we may need to re-name `MaxTimestamp` to `UnsetTimestamp` and leave that as `math.MaxUint64` then make `MaxTimestamp` be `MaxInt64`.